src/features/draft/commands/handlers/update-row.handler.ts

```diff
@@ -65,6 +84,7 @@ export class UpdateRowHandler extends DraftHandler<
         rowId: data.rowId,
         data: await this.pluginService.afterUpdateRow(data),
         schemaHash,
+        publishedAt,
```

Comment:
The best way would be to emphasize that we only save publishedAt if publishedAt is not equal to `undefined`.

maybe `getFirstPublishedAtFromDataOrUndefined?`

```
publishedAt: await this.getFirstPublishedAtFromDataOrUndefined( ... )
```